### ISSUE TITLE
lmstudio: build fails with hash-mismatch

### ISSUE THREAD

**Issue by aviallon** [NixOS/nixpkgs#315452]
```console
[user@nixos]$ NIXPKGS_ALLOW_UNFREE=1 nix shell --impure github:NixOS/nixpkgs/master#lmstudio
error: hash mismatch in fixed-output derivation '/nix/store/ckijj426ghwa7aj1cr0zba8lkn0z6kf4-LM_Studio-0.2.22.AppImage.drv':
         specified: sha256-hcV8wDhulFAxHDBDKicpEGovwcsn9RaIi/idUz+YzD8=
            got:    sha256-HoAVUU5Z5ueGMSybhT0OKV611v1yU5am0wy0GsCUmvk=
```
https://github.com/NixOS/nixpkgs/blob/bfb7a882678e518398ce9a31a881538679f6f092/pkgs/by-name/lm/lmstudio/linux.nix#L11

**Comment by eclairevoyant** [comment]
No hash mismatch on my end, and I've never built nor heard of lmstudio before. (i.e. there's no caching involved here)

**Comment by jee-mj** [comment]
```sh
 mj@URIEL-WS  ~/dotFiles  sudo nixos-rebuild boot --flake ~/dotFiles
building the system configuration...
trace: warning: cudaPackages.autoAddDriverRunpath is deprecated, use pkgs.autoAddDriverRunpath instead
error: hash mismatch in fixed-output derivation '/nix/store/2b5p9x00myfkpaill9dk8qviyjma3hkq-LM_Studio-0.2.22.AppImage.drv':
         specified: sha256-hcV8wDhulFAxHDBDKicpEGovwcsn9RaIi/idUz+YzD8=
            got:    sha256-HoAVUU5Z5ueGMSybhT0OKV611v1yU5am0wy0GsCUmvk=
error: 1 dependencies of derivation '/nix/store/46f5lrm3b40mi42vi0aiydbbw5vmzirz-lmstudio-0.2.22-extracted.drv' failed to build
error: 1 dependencies of derivation '/nix/store/a0p284bbsmx7spkmnjc83419r5i51zyq-lmstudio-0.2.22.drv' failed to build
error: 1 dependencies of derivation '/nix/store/ldwd6i79ml6mb0m2w02p4jnsrhv8r5w1-user-environment.drv' failed to build
error: 1 dependencies of derivation '/nix/store/38b9s372nvvwvb4nmjp8qz8l8afwprc5-etc.drv' failed to build
error: 1 dependencies of derivation '/nix/store/09p5g89f81gqwhh44is3d1sxk44ylkjl-nixos-system-URIEL-WS-24.11.20240529.ad57eef.drv' failed to build
```
I am getting errors building when I add the package to my [flake => (configuration.nix, home-manager) dotfiles](https://github.com/jee-mj/dotFiles)...

**Comment by eclairevoyant** [comment]
Your `flake.lock` is missing, so we don't know what revision you're on atm

**Comment by JustinLex** [comment]
Fixed in #314992. The PR has not reached unstable yet: https://nixpk.gs/pr-tracker.html?pr=314992

See discussion in #310914 on why the hash mismatch happened.
